main.rb

```diff
@@ -0,0 +1,86 @@
+require_relative 'classes/item'
+require_relative 'app'
+require_relative 'utils/book_handler'
```

Comment:
- You have done great so far :+1: :100: But there are a few requirements you need to meet before moving to the next milestone. :+1: :+1: 
- - Error screenshot:

![Selection_060](https://github.com/Kingno3l/Ruby-Capstone/assets/107632780/97377209-bb88-4c46-9b96-81053b40cfa1)

- [ ] While trying to start the app, I got this error shown in the screenshot above, and this is because of the incorrect casing of the require statement above. the `u` in `require_relative 'utils/book_handler' should be a capital letter. I suggest you rectify this. :+1: :100:

Reply:
@V-Blaze the app starts perfectly on my machine without implementing the changes you suggested. Hope you were on the development branch. Below is the screen shoot 
![Screenshot (2)](https://github.com/Kingno3l/Ruby-Capstone/assets/69603174/1ce96441-6e6e-405b-9e00-acb391668fde)